### DISCUSSION TITLE
Add a check point

### DISCUSSION THREAD

**Discussion by ShengjieXu667** [jspsych/jsPsych#1245] [Q&A] [answered]
I want to add a check point in my practice block, and if the accuracy of current subject does not reach 80%, then s/he needs to re-do the practice until passes 80%. I tried some approaches but they did not work. I wonder how to implement this in jspsych? Thanks in advance!

**Comment by becky-gilbert** [comment, marked as answer]
Hi @ShengjieXu667, I think this is covered in the [Moving Research Online "Advanced jsPsych" YouTube workshop, starting around 45 min in](https://youtu.be/LP7o0iAALik?t=2720). You can also have a look at [the final code from that tutorial](https://github.com/jspsych/moving-research-online-workshop/blob/master/day-3-semantic-relatedness/semantic_relatedness_finish.html). 

I hope that helps! Feel free to post your code if you run into problems or need further guidance.

**Reply by ShengjieXu667** [reply]
Thanks Becky, it works!
